Sources/swift-build/usage.swift

```diff
@@ -26,6 +26,8 @@ func usage(_ print: (String) -> Void = { print($0) }) {
     print("  --fetch                        Fetch package dependencies")
     print("  --update                       Update package dependencies")
     print("  --generate-xcodeproj[=<path>]  Generates an Xcode project [-X]")
+    print("  --show-dependencies[=<mode>]   Print dependency graph (text|dot|json)")
```

Comment:
Normally I would put it into a separate PR, but it's just one line and would conflict with the line I'm adding. So I just sneaked it in and mentioned it in the PR description.